data_store/table_extraction/process.py

```diff
@@ -75,38 +76,45 @@ def process(self, table: Table) -> None:
         :param table: The table to process.
         :return: None
         """
+
         self._strip_whitespace(table)
         self._lift_header(table)
         self._remove_merged_headers(table)
         self._drop_cols_by_name(table)
         self._remove_ignored_non_header_rows(table)
         self._replace_dropdown_placeholder(table)
         self._drop_bespoke_rows(table)
+
         if self.drop_empty_rows:
             self._drop_empty_rows(table)
+
         if self.drop_empty_tables and table.df.empty:
-            table.df = None
+            table.df = None  # type: ignore[assignment]
```

Comment:
@samuelhwilliams Ignoring it. At least for now. Reverted the tests too.

I don't think the `| None` would be the good approach though. 

I think in theory we either make `pd.DataFrame` always work, or we delete it completely, if unused.

Reply:
Look at yourself in the mirror Albert. Look who you've become.